app/forms/select_email_form.rb

```diff
@@ -7,6 +7,9 @@ class SelectEmailForm
   attr_reader :user, :identity, :selected_email_id
 
   validate :validate_owns_selected_email
+  validates :selected_email_id, presence: {
+    message: proc { I18n.t('email_address.not_found') },
+  }
```

Comment:
Can you add test coverage for this to `spec/forms/select_email_form_spec.rb` ?

Reply:
Added in with [caf53d2](https://github.com/18F/identity-idp/pull/11568/commits/caf53d26645483bd5ecbd13df5e2ee6215532685)